lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -20,10 +20,16 @@ const CriticalRequestChainRenderer =
     require('../../../../report/html/renderer/crc-details-renderer.js');
 
 const sampleResultsStr = fs.readFileSync(__dirname + '/../../../results/sample_v2.json', 'utf-8');
-const sampleResultsRoundtripStr = fs.readFileSync(
-  __dirname + '/../../../../../proto/sample_v2_round_trip.json',
-  'utf-8'
-);
+
+let sampleResultsRoundtripStr;
+try {
+  sampleResultsRoundtripStr =
```

Comment:
it's weird that only one test in this file is using `sampleResultsRoundtripStr` even though the PSI renderer will *only* be getting lhrs round-tripped through a proto...

Reply:
@exterkamp is this something you'd like to fix?